eppo/build.gradle

```diff
@@ -68,7 +68,7 @@ ext.versions = [
 ]
 
 dependencies {
-    api 'cloud.eppo:sdk-common-jvm:3.7.0'
+    api 'cloud.eppo:sdk-common-jvm:3.7.1-SNAPSHOT'
```

Comment:
Release after common v3.7.1 is available

Reply:
3.7.1 is now live https://repo1.maven.org/maven2/cloud/eppo/sdk-common-jvm/3.7.1/